pages/1.9/release-notes/1.9.8/index.md

```diff
@@ -0,0 +1,235 @@
+
+[preview]
+#### GPU
+[/preview]
+- Leverage GPUs to run novel algorithms.
+- Because DC/OS GPU support is compatible with nvidia-docker, you can test locally with nvidia-docker and then deploy to production with DC/OS.
+- Allocate GPUs on a per container basis, including isolation guarantees
+
+For more information, see the [documentation](/1.9/deploying-services/gpu/).
+
+## DC/OS Monitoring and Operations
+
+[preview]
+### Remote Process Injection for Debugging
+[/preview]
+
+The new `dcos task exec` command allows you to remotely execute a process inside the container of a deployed Mesos task, providing the following features.
+
+- An optional `--interactive` flag for interactive sessions.
+- Attach to a remote pseudoterminal (aka PTY) inside a container via the optional `--tty` flag.
+- Combine the `--interactive` and `--tty` flags to launch an interactive bash session or to run `top` and see the resource usage of your container in real time.
+
+For more information, see the debugging [documentation](/1.9/monitoring/debugging/).
+
+[preview]
+### Logging
+[/preview]
+
+Stream task and system logs to journald by setting the `mesos_container_log_sink` install-time parameter to `journald` or `journald+logrotate`. This allows you to:
+
+- Include task metadata like container ID in your queries to more easily locate the logs that you want.
+- Use the new DC/OS CLI commands `dcos node log` and `dcos task log` to query logs. You can also make HTTP requests directly against the new Logging API.
+- Set up log aggregation solutions such as Logstash to get logs into their aggregated storage solutions.
+
+For more information, see the [documentation](/1.9/monitoring/logging/).
+
+[preview]
+### Metrics
+[/preview]
+
+- Node-level HTTP API that returns metrics from tasks, cgroup allocations per container, and host level metrics such as load and memory allocation.
+- StatsD endpoint in every container for forwarding metrics to the DC/OS metrics service. This service is what exposes the HTTP API.
+- Any metric sent to STATSD_UDP_HOST/PORT is available in the HTTP API’s `/container/<container_id>/app` endpoint.
+
+For more information, see the [documentation](/1.9/metrics/).
+
+### Tool for Troubleshooting Service Deployment Failures
+
+- The new service deployment troubleshooting tool allows you to find out why your applications aren’t starting from the GUI and CLI.
+
+  ![Service deploy GUI](/1.9/img/dcos-offers.png)
+
+### Improved GUI
+
+- New look and feel and improved navigation.
+
+  ![New GUI](/1.9/img/dcos-dash.png)
+
+- Usability improvements to the service create workflow.
+
+  ![Improved GUI](/1.9/img/dcos-create.png)
+
+## Networking Services
+
+- CNI support for 3rd party CNI plugins.
+- Performance improvements across all networking features.
+
+## Security and Governance
+
+- DC/OS Identity and Access Management (IAM) highlights:
```

Comment:
I think that all the features in this section might be enterprise (and therefore need enterprise tags). Please check on it though, because I'm not entirely sure.

Reply:
@judithpatudith what is the next step? Did you hear back from PMs on the above question?

Reply:
yes, the enterprise tags are correct.

Reply:
@somikbehera your answer is not clear. Could you please review the list of features in 1.9 (sections under : About DC/OS 1.9) release and indicate which is for ENTERPRISE only?

Reply:
All of the features in the section `DC/OS Identity and Access Management (IAM) highlights:` are enterprisy.

Reply:
More generally, all of the things mentioned in the section `Security and Governance` are enterprisy. I double-checked this for every single line in this section.

Reply:
Added Enterprise tags for each feature in "Security and Governance" section.